<!DOCTYPE html>
<html>
	<head>
		<link rel="stylesheet" href="dist/css/bootstrap.css">
		<link href="Shift_font.css" rel="stylesheet">
		<link type="text/css" rel="stylesheet" href="stylesheet.css"/>
		<title>Sumitabha Brahmachari-About Me</title>
	</head>
	<body>
		
		<div class="jumbotron center" >
			<div class="container">
                <h1>Sumitabha Brahmachari</h1>
            </div>

		</div>

		<div class="navbars">
			<div class="container">
				<ul align="center" class="nav nav-tabs">
					<li> <a href="index.html"> About</a>  </li>
					<li class="active"> <a href="research.html"> Research</a> </li>
					<li> <a href="cv.html"> CV</a>  </li>
					
				</ul>
			</div>
		</div>



		
		<div class="homebody">
			<div class="container">
				<p> 
					<h3>
						Genome Organization 
					</h3>
					
					<h4>
						The long genome (about one meter!) undergoes severe compaction (about a hundred thousand fold!) 
						to fit inside the nucleus (about ten microns). 
						One may be tempted to attribute this as a classic case of a collapsed polymer globule. 
						However, many observations, e.g., the genome contains chromosomes (typically, tens in number) that
						occupy separate territories, and chromosomes harbor open and closed compartments, suggest that the reality is more complex.
						The current notion is that the genome is organized in a hierarchical fashion. 
						I have used coarse-grained polymer modeling to elucidate how the collective action of various proteins constantly interacting
						with the genome may be simplified into three generic forces.
						Competition between these forces may bring about drastic, yet predictable changes in the genomic structure, many of 
						which characterize the species-wide chromosome organization.
						
					
						<ul>
							<li>
								<a href="#">Shaping the Genome via Lengthwise Compaction, Phase Separation, and Lamnina Adhesion </a>
								S. Brahmachari, V. Contessoto, M. Di Pierro, and J.N. Onuchic. <em>BioRxiv</em> (2022).
							<a href="pdfs/Brahmachari_biorxiv_2022.pdf">[PDF]</a>
							</li>

							<li>
								<a href="https://doi.org/10.1126/science.abe2218">3D Genomics Across the Tree of Life Identifies Condensin II as a Determinant of Architecture Type.</a>
								C. Hoencamp*, A.M.O Elbatsh*, O. Dudchenko*, S. Brahmachari*, et. al. 
								<em>Science</em> 372 6545 (2021).
								<a href="pdfs/Hoencamp-etal-Science2021.pdf">[PDF]</a>
							</li>

							
							<li>
								<a href="https://doi.org/10.1073/pnas.1906355116">Chromosome Disentanglement Driven Via Optimal Compaction of Loop Extruded-Brush Structures.</a>
								S. Brahmachari and J.F. Marko, 
								<em>Proc. Natl. Acad. Sci. USA</em> 116, 24956 (2019).
								<a href="pdfs/Brahmachari-Marko-PNAS2019.pdf">[PDF]</a>							
						</ul>
					</h4>
				</p>
				<img height="400" src="images/genome.png" style="border:1px solid white; float:center; object-fit:cover;"/>
				

				<p>
					<br><br><br>
					<h3>
					Statistical Mechanics of DNA and Chromatin
					</h3>
					<h4>
						DNA mechanics is central to protein-DNA interactions.
						Proteins binding to the DNA groove bend and twist DNA leading to buildup of torsion or tension along the DNA chain.
						The mechanical response of DNA to the torsion or force could bring about large scale reorganization, 
						like the formation of plectonemes.
						Characterizing the mechanical response of DNA to torque and force is important to understand 
						how proteins manipulate DNA struture in the cell.
					<ul>
						<li>
							<a href="https://doi.org/10.1093/nar/gkaa836">Coarse-Grained Modeling of DNA Plectoneme Formation in the Presence of Base-Pair Mismatches.</a>
							P.R. Desai, S. Brahmachari, J.F. Marko, S. Das, and K.C. Neuman.
							<em>Nucl. Acids Res.</em> 48 19:10713-10725 (2020).
							<a href="pdfs/Desai-etal-NAR2020.pdf">[PDF]</a>
						</li>

						<li>
							<a href="https://doi.org/10.1007/978-3-319-95294-9_2">DNA Mechanics and Topology.</a>
							S. Brahmachari, and J.F. Marko.
							<em>Biomechanics in Oncology</em>,
								(Edited by C. Dong, N. Zahir, and K. Konstantopoulos; Springer International Publishing, 2018) pp. 11-39.
							<a href="pdfs/Brahmachari-Marko-chapter2018.pdf">[PDF]</a>
							
						</li>
						

						<li>
							<a href="https://doi.org/10.1103/PhysRevE.97.022416">Defect-Facilitated Buckling in Supercoiled Double-Helix DNAs.</a>
							S. Brahmachari, A. Dittmore, Y. Takagi, K.C. Neuman, and J.F. Marko.
							<em>Phys. Rev. E</em> 97, 022416 (2018).
							<a href="pdfs/Brahmachari-etal-PRE2018.pdf">[PDF]</a>	
						</li>

						<li>
							<a href="https://doi.org/10.1103/PhysRevLett.119.188103">Nucleation of Multiple Buckled Structures in Intertwined DNA Double Helices.</a>
							S. Brahmachari*, K.H. Gunn*, R.D. Giuntoli, A. Mondragon, and J.F. Marko. 
							<em>Phys. Rev. Lett.</em> 119, 188103 (2017). 
							<a href="pdfs/Brahmachari-etal-PRL2017.pdf">[PDF]</a>
						</li>
			
						<li>
							<a href="https://doi.org/10.1103/PhysRevLett.119.147801">Supercoiling Locates Mismatches.</a>
							A. Dittmore, S. Brahmachari, Y. Takagi, J.F. Marko, and K.C. Neuman. 
							<em>Phys. Rev. Lett.</em> 119, 147801 (2017).
							<a href="pdfs/Dittmore-etal-PRL2017.pdf">[PDF]</a>
						</li>
			
						<li>
							<a href="https://doi.org/10.1103/PhysRevE.95.052401">Torque and Buckling in Stretched Intertwined Double-Helix DNAs.</a>
							S. Brahmachari and J.F. Marko.
							<em>Phys. Rev. E</em> 95, 052401 (2017).
							<a href="pdfs/Brahmachari-Marko-PRE2017.pdf">[PDF]</a>
							
						</li>
						
						

						
					</ul>
					</h4>
					<img height="400" src="images/DNAmech.png" style="border:1px solid white; float:center; object-fit:cover;"/>
					
					<br><br><br>
					<h3>
						Gene Transcription 
					</h3>

					<h4>
						The trancribing motion of RNA polymerases 
						twist DNA, can the twist act as a mechanical regulator of gene trannscription? We find that DNA supercoiling 
						can mediate long-range interaction between co-transcribing RNA polymerases. 

					
						<ul>
							<li>
								<a href="https://doi.org/10.1093/nar/gkab1252">
									DNA Supercoiling-Mediated Collective Behavior of Co-Transcribing RNA Polymerases.
								</a>
							S. Tripathi, S. Brahmachari, J.N. Onuchic, and H. Levine.  
							<em>Nucl. Acids Res.</em> gkab1252 (2021).
									<a href="pdfs/Tripathi-etal-NAR2021.pdf">
								[PDF]
							</a>
							</li>
						</ul>
						<img height="300" src="images/gene.png" style="border:1px solid white; float:center; object-fit:cover;"/>
					</h4>
				</p>
				<br><br>
				<br><br>
				<h3>
					Active polymers
				</h3>
				<h4>
					Activity in a coarse-grained system can manifest itself in many different ways. 
					A common characteristic of activity is a non-white (colored) noise spectrum, which breaks detailed balance, the hallmark of equilibrium.
					I am interested in simulating polymers with a temporally correlated noise, to explore the signatures of activity
					that may be observed in experiments. One such signature is an increased time exponent in the mean-squared 
					displacement versus time plots.

				</h4>
				<img height="250" src="images/active.png" style="border:5px solid white; float:center;"/>
				<img height="250" src="images/active-msd.png" style="border:5px solid white; float:center;"/>
				<br><br>
				<br><br>

			
			</div>

			
		</div>

	</body>
</html>
